### PR TITLE
fix referral link, select coin bug

### PR DESCRIPTION
**Pull request by erwin-huang**

### CODE REVIEW

File: src/state/swap/hooks.ts [hunk 1]
```diff
@@ -301,18 +301,19 @@ export function useDefaultsFromURLSearch():
   useEffect(() => {
     if (!chainId) return
     const parsed = queryParametersToSwapState(parsedQs)
+    const outputCurrencyId = parsed[Field.OUTPUT].currencyId ? parsed[Field.OUTPUT].currencyId : KODA.address
```
Comment (LukeSamkharadze) [inline src/state/swap/hooks.ts:304]: You can use `??` operator

Comment (erwin-huang) [reply src/state/swap/hooks.ts:304]: when `outputCurrencyId = ""`
I think it won't work if I use `??` operator 

image

Comment (LukeSamkharadze) [reply src/state/swap/hooks.ts:304]: Oh, what about  || then?

**Review by LukeSamkharadze — APPROVED** [review]

File: src/state/swap/hooks.ts (hunk 1 above)
Comment (erwin-huang) [reply src/state/swap/hooks.ts]: Nice, It's working :ultra_fast_parrot:

**Pull request merged**